dist/practice.js

```diff
@@ -3204,7 +3220,7 @@ var SIM_CONTROLLER = (function () {
 				// Check Nullify
 				if (target.nullified && !skill.ignore_nullify) {
 					target.nullified--;
-					if (debug) echo += debug_name(src_card) + ' wing guards ' + debug_name(target) + ' but it is nullified!\n';
+					if (debug) echo += debug_name(src_card) + ' wing guards ' + debug_name(target) + ' but it is nullified!<br>';
```

Comment:
💚 Thanks for cleaning up this log message.  Good catch on the incorrect use of `\n` over `<br>`.

Normally I'd suggest putting this in a separate PR, as it's unrelated to the main purpose of this one, but it's smaller enough to not require its own PR.

Reply:
I pushed the incorrect newline in one of the first PRs and didn't notice it (oops).
Got it about using separate PRs.